test/WebJobs.Script.Tests/CSharpEndToEndTests.cs

```diff
@@ -32,6 +32,25 @@ public CSharpEndToEndTests(TestFixture fixture) : base(fixture)
         }
 
         [Fact]
+        public async Task NotificationHub()
+        {
+            await NotificationHubTest();
+        }
+
+        [Fact]
+        public async Task NotificationHub_Out_Notification()
+        {
```

Comment:
I'm wondering about the scenario where someone git pushes their functions with a project.json, to a site where the host is not running (e.g. new site creation). When it starts up, i'd expect stuff to just work. @fabiocav?